### ISSUE TITLE
Whether to add model eval()?

### ISSUE THREAD

**Issue by XubeiPan666** [haofanwang/video-swin-transformer-pytorch#9]
Hello, I tried it if there is no model.eval(),  which results in the same data input but get different feature output. I think this is because of the dropout layer in the model, so I think whether to add model.eval() to ensure that the same input gets the same output.

**Comment by haofanwang** [comment]
You're right. In practice, you will need to set the model to eval mode during testing.
